department-of-veterans-affairs/vets-website
# Add marketable land question

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

On Step 5 of Financial Information: income,
- create a new page for the "Is the additional land marketable?" question
- Include supporting copy that provides examples
- match all copy to designs

## How to run in local environment
- Check out this branch locally
- In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
- Run vets-website and vets-api

## How to verify
- Go to http://localhost:3001/pension/application/527EZ in your browser
- Select 'Start your application without signing in'
- Proceed to Chapter 5 by filling out the required fields in previous chapters
- On the question, "Is the additional land marketable?", confirm that the page matches the designs

## Related issue(s)

- [Link to github ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70428)

- [Link to epic](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/70428)

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |   N/A (new question)     |   ![image](https://github.com/department-of-veterans-affairs/vets-website/assets/54864006/dae15329-356a-46fc-b981-202c73fe3adc)    |
| Desktop |    N/A (new question)    |   ![image](https://github.com/department-of-veterans-affairs/vets-website/assets/54864006/d70e1010-82b2-45ff-bf10-3a227feeb1b6)    |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
